server/database/seeds.js

```diff
@@ -1,5 +1,14 @@
+const fs = require('fs');
 const makeSchema = fs.readFileSync('server/database/schema.sql').toString();
-const connection = require('./query.js');
+const mysql = require('mysql');
+const credentials = require('./config.js');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: credentials.username,
+  password: credentials.password,
+  database: credentials.database,
+  multipleStatements: true,
+});
 
 connection.query(makeSchema, (error) => {
```

Comment:
This is the part that I had difficulty getting to work on my machine. If you also encounter issues with this down the road, I would suggest maybe keeping the schema.sql separate from the seeder file and just setting up scripts in your package.json or in your docker container to run each process sequentially.